app-service-environment-serviceplan-logicapp/azuredeploy.json

```diff
@@ -8,6 +8,12 @@
         "description": "The name to use for the new AppService Environment"
       }
     },
+    "location": {
```

Comment:
using resourceGroup().location is a better practice here, is there something specific we need to break it out into a param for?